cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -80,6 +80,7 @@
     cronjob at:'30 6 * * *', do:deploy_dir('bin', 'cron', 'admin_progress')
     cronjob at:'10 7 * * *', do:deploy_dir('bin', 'cron', 'retention_stats')
     cronjob at:'*/1 * * * *', do:deploy_dir('aws', 'ci_build'), notify: 'dev+build@code.org'
+    cronjob at:'@reboot', do:dashboard_dir('bin', 'sqs', 'process_queues_daemon restart')
```

Comment:
Looking at the syntax of `process_queues_daemon`, doesn't this take an `-r` option, like `process_queues_daemon -r restart`?

also, could we use `start` rather than `restart` on this since this will only be run on reboot? Running `restart` will throw a warning if the service is not already started, which I think might cause email/honeybadger alert to be triggered.